run.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-java -cp "lib/*:." ConnectionCheck $@
+java -cp "lib/jcommander-1.82.jar" connectionTool.Main $*
```

Comment:
line ending is still incorrect,
the other jars and current dir are missing:
gpawel@pm-ubuntu2:~/db-connection-check$ ./run.sh
Error: Could not find or load main class connectionTool.Main